apps/frontend/src/app/PageTeams/index.tsx

```diff
@@ -56,8 +56,64 @@ export default function PageTeams() {
       return
     }
   }
+  const [teamDisplayState, setteamDisplayState] = useState(
+    database.displayTeam.get()
+  )
+
+  // I can't process why thid is not working
```

Comment:
This doesnt work because the `useEffect` dependency array checks changes in a value. However, the value of `database.displayTeam` does not change, due to the fact that it is referencing the same object.

Reply:
I tried doing it like this
```ts
  useEffect(
    () => setteamDisplayState(database.displayTeam.get()),
    [
      database.displayTeam,
      database.displayTeam.data.sortType,
      database.displayTeam.data.ascending,
    ]
  )
```
and still didn't work.

So I worked around it by setting the state in `onChange` and `onChangeAsc` functions.

Reply:
Ignore this I just noticed `useDataEntryBase` create a state and updates it all on its own 👍.